src/components/controls/textbox/Textbox.js

```diff
@@ -91,13 +92,17 @@ class Textbox extends React.Component {
     super(props);
 
     this.state = {
-      currentValue: ''
+      currentValue: props.value
```

Comment:
Problem is using ``value`` as the prop name. it's conflicting with the HTML ``value`` attribute. I've changed the name to ``initialValue`` and the problem is now resolved. I made this a minor instead of a patch because ``value`` will still work, but ``initialValue`` is more correct and how people should be using this component in the future.